### PR TITLE
Adds more brain damage quotes

### PR DESCRIPTION
**Pull request by hal9000PR**
## What Does This PR Do
adds more quotes for when you are brain damaged

## Why It's Good For The Game
More variety, some funny memes for stupid brain hurt people to say. Currently the list isn't reaching its full potential 

## Changelog
:cl:
add: added more brain damage quotes
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/life.dm [hunk 1]
```diff
@@ -133,10 +133,17 @@
...
 							   "the CHef put [pick("PROTEIN", "toiret waTer", "RiPPleing TendIes", "Einzymes","HORRY WALTER","nuTriments","ReActive MutAngen","TeSLium","sKrektonium")] in my [pick("wiSh soup","Bullito","rAingurber","sOilent GREEn","KoI Susishes","yaya")]!",
 							   "the monkey have TASER ARMS!",
-							   "qM blew my points on [pick("cOMbat Shtogun","inSuLated gloves","LOTS MASSHEEN!")]",
+							   "qM blew my points on [pick("cOMbat Shtogun","inSuLated gloves","LOTS MASSHEEN!", "peNTa Smmmmmmmm")]",
 							   "EI'NATH!",
 							   "WAKE UP SHEEPLES!",
-							   "et wus my [pick("wittle brother!!","fiancee","friend staying over","entiRe orphanage","love interest","wife","husband","liTTle kids","sentient cAT","accidentally")]!")
+							   "et wus my [pick("wittle brother!!","fiancee","friend staying over","entiRe orphanage","love interest","wife","husband","liTTle kids","sentient cAT","accidentally")]!",
+							   "magnazin is just an plate fur gun foond",
+							   "WHERE RECHAN BATONG?",
```
Comment (S34NW) [inline code/modules/mob/living/carbon/human/life.dm:141]:
```suggestion
							   "WHERE I CHARGE BATONG?",
```

image
just so the reference is more closely followed

File: code/modules/mob/living/carbon/human/life.dm [hunk 2]
```diff
@@ -133,10 +133,17 @@
...
+							   "HELP I'M MANTIS!!",
```
Comment (S34NW) [inline code/modules/mob/living/carbon/human/life.dm:142]:
```suggestion
							   "HELP MANTIS!!",
```
I'm seems a bit nonsensical

**Review by S34NW — COMMENTED** [review]
mostly fine, atrocious spelling and grammar but that _is_ the point... So I'll let it pass.

File: code/modules/mob/living/carbon/human/life.dm (hunk 2 above)
Comment (hal9000PR) [reply code/modules/mob/living/carbon/human/life.dm:142]: counter point
image

Comment (Cocacolagua) [reply code/modules/mob/living/carbon/human/life.dm:142]: :trollface:

Comment (S34NW) [reply code/modules/mob/living/carbon/human/life.dm:142]: image

Comment (S34NW) [reply code/modules/mob/living/carbon/human/life.dm:142]: counterpoint accepted

File: code/modules/mob/living/carbon/human/life.dm [hunk 3]
```diff
@@ -133,10 +133,17 @@
 							   "my dad own this station",
 							   "the CHef put [pick("PROTEIN", "toiret waTer", "RiPPleing TendIes", "Einzymes","HORRY WALTER","nuTriments","ReActive MutAngen","TeSLium","sKrektonium")] in my [pick("wiSh soup","Bullito","rAingurber","sOilent GREEn","KoI Susishes","yaya")]!",
 							   "the monkey have TASER ARMS!",
-							   "qM blew my points on [pick("cOMbat Shtogun","inSuLated gloves","LOTS MASSHEEN!")]",
+							   "qM blew my points on [pick("cOMbat Shtogun","inSuLated gloves","LOTS MASSHEEN!", "peNTa Smmmmmmmm")]",
```
Comment (AffectedArc07) [inline code/modules/mob/living/carbon/human/life.dm:136]: This or pod tesla. Pick one self-meme reference.

**Review by AffectedArc07 — CHANGES_REQUESTED** [review]

**Comment by TheYeetster1** [comment]
Help I have dain bramage.

**Review by Kyep — COMMENTED** [review]
None of these memes seem particularly funny.
Certainly not at the level the others listed.

**Pull request closed without merging**